packages/fether-react/src/Send/TxForm/TxForm.js

```diff
@@ -129,6 +129,22 @@ class TxForm extends Component {
       token
     } = this.props;
 
+    if (!chainId) {
+      throw new Error('chaindId is required for an EthereumTx');
+    }
+
+    if (!address) {
+      throw new Error('address of an account is required');
+    }
+
+    if (!transactionCount) {
+      throw new Error('transactionCount is required for an EthereumTx');
```

Comment:
This will make the app crash, right? there's no error catcher here if i'm not mistaken. Should we put these throws inside validateForm (or prevalidate) instead?

Reply:
Ok, I've pushed a commit moving them into `validateForm` function